kubetest2-tf/pkg/tfvars/tfvars.go

```diff
@@ -8,8 +8,8 @@ type TFVars struct {
 	StorageBucket  string `json:"bucket,omitempty"`
 	StorageDir     string `json:"directory,omitempty"`
 	ClusterName    string `json:"cluster_name"`
-	ApiServerPort  int    `json:"apiserver_port"`
-	WorkersCount   int    `json:"workers_count"`
+	ApiServerPort  int    `json:"apiserver_port,string"`
```

Comment:
This is done to allow the `maps.Insert` function to type match against adding elements to map[string]string.

Reply:
Switched to `string`, along with the supporting changes wherever required in the ansible/terraform side.